src/libs/ajax.js

```diff
@@ -1113,6 +1113,36 @@ const Clusters = signal => ({
   }
 })
 
+const Apps = signal => ({
+  list: async (project, labels = {}) => {
+    const res = await fetchLeo(`api/google/v1/apps/${project}/?${qs.stringify({ ...labels })}`,
```

Comment:
nit: Leo seems to accept this, but technically there should not be a `/` at the end here.
```suggestion
    const res = await fetchLeo(`api/google/v1/apps/${project}?${qs.stringify({ ...labels })}`,
```